traitsui/qt4/table_editor.py

```diff
@@ -1251,3 +1265,11 @@ def _update_filter_list(self):
         filters = self.filters
         self.filters = []
         self.filters = filters
+
+if is_qt5:
```

Comment:
Minor style suggestion: put this at the top of the file (possibly immediately after the definition of `is_qt5`).